data/json/mutations/mutations.json

```diff
@@ -7075,8 +7075,8 @@
     "category": [ "PLANT" ],
     "attacks": [
       {
-        "attack_text_u": "You lash %s with a vine",
-        "attack_text_npc": "%1$s lashes %2$s with their vines",
+        "attack_text_u": "You lash %s with a vine!",
```

Comment:
Why does the player character use a single vine to attack while NPCs use multiple? Minor inconsistency. Same in line 7106.